kahuna/public/js/preview/image.js

```diff
@@ -106,13 +106,31 @@ image.controller('uiPreviewImageCtrl', [
     ctrl.srefNonfree = () => storage.getJs("isNonFree", true) ? true : undefined;
 
     ctrl.hasActiveAllowLease = ctrl.image.data.leases.data.leases.find(lease => lease.active && lease.access === 'allow-use');
+
+    ctrl.showOverlay = (image) => $window._clientConfig.enableWarningFlags && ctrl.isSelected && (ctrl.showAlertOverlay(image) || ctrl.showWarningOverlay(image));
```

Comment:
Is there a reason we are passing the image here, rather than using ctrl.image? I can't see these functions being called with anything other than the image that is currently in scope.

Reply:
I see you have valid point @jonathonherbert. I have removed the image param since they are fixed anyway